config/locales/en/activerecord/course/assessment/category.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  activerecord:
+    errors:
+      models:
+        course/assessment/category:
+          deletion: 'The last category cannot be deleted'
```

Comment:
strictly speaking, this should be a sentence fragment, so probably 'cannot be deleted'